CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Stryker Dashboard
```

Comment:
I'm missing some docs here on how to run the project. I think that's the most important part of a contributing.md. What tools do I need, and what do I need to do to contribute for each project

Reply:
Yeah that works for me. I think the instructions in the readme could use a little love either way

Reply:
Agreed, I'll pick that up in a different PR though (#565)